go.mod

```diff
@@ -100,6 +100,7 @@ require (
 	github.com/openziti/secretstream v0.1.17 // indirect
 	github.com/openziti/transport/v2 v2.0.124 // indirect
 	github.com/orcaman/concurrent-map/v2 v2.0.1 // indirect
+	github.com/panjf2000/ants v1.3.0
```

Comment:
it's a new dependency, so we need the review from TSC
it should be fine
However, you need to put this go modudle to the normal section, not the indirect section
Why don't you use the latest release verson?
https://github.com/panjf2000/ants/releases/tag/v2.9.1

You need to update the Attribution.txt for this new dependency as well
https://github.com/edgexfoundry/device-sdk-go/blob/97b36af7baff2da76e29dc2359062fa76ab64ed3/example/cmd/device-simple/Attribution.txt